package.json

```diff
@@ -64,5 +64,7 @@
       "**/hardhat/**"
     ]
   },
-  "dependencies": {}
+  "dependencies": {
+    "react-confetti": "^6.0.1"
```

Comment:
This should be moved to inside the react-app directory package.json since its only used for the front end